knoten/warp.py

```diff
@@ -0,0 +1,71 @@
+import jinja2
+
+def generate_vrt(raster_size, gcps, fpath,
```

Comment:
I'd rather call this to_vrt or something else to indicate it's writing out a file.